src/mantarray_desktop_app/mantarray_front_panel.py

```diff
@@ -30,6 +36,37 @@ def read_wire_out(self, ep_addr: int) -> int:
             )
         return 0
 
+    def activate_trigger_in(self, ep_addr: int, bit: int) -> None:
+        # pylint: disable=no-self-use,unused-argument # Tanner (7/14/20): this method should never actually be used by child classes. They should use this method provided by the other parent class due to MRO.
+        if isinstance(self, FrontPanelBase):
+            raise NotImplementedError(
+                "This implementation of activate_trigger_in should never be called by a FrontPanelBase object. Use a FrontPanelBase implementation instead"
+            )
+
+    def clear_barcode_scanner(self) -> None:
+        self.activate_trigger_in(
+            BARCODE_SCANNER_TRIGGER_IN_ADDRESS, CLEAR_BARCODE_TRIG_BIT
+        )
+
+    def start_barcode_scan(self) -> None:
+        self.activate_trigger_in(
+            BARCODE_SCANNER_TRIGGER_IN_ADDRESS, START_BARCODE_SCAN_TRIG_BIT
+        )
+
+    def get_barcode(self) -> str:
+        """Convert 3 32-bit wire out values to a barcode string."""
```

Comment:
No, I was thinking of doing that but realized it could potentially get too complicated deciding when to trim a `0x00` byte since that is the cleared wireout value and we actually want to see that value. I thought it made more sense for this function to just convert the wireout values to a string and let OKComm handle deciding what is valid/invalid/empty/ etc.